server/routes/userController.js

```diff
@@ -167,4 +151,13 @@ router.post("/edit", authCheck, function (req, res) {
   return res.send("Editing File");
 });
 
-module.exports = router;
+router.post("/itemLists/addLists", authCheck, async (req, res) => {
```

Comment:
You'll want to use more conventional naming for these endpoints https://restfulapi.net/resource-naming/